### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Reflow:Text are getting truncated at 400% zoom.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/vscode-docs#5104]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Reflow;#WCAG1.4.10; #FTP; #ChromiumEdge;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Pre-requisites:**
Set the Display Resolution : 1280*1024.

**Repro Steps:**

1. Launch the application Visual Studio Code Service.
2. Observe that text are truncated at 400% zoom

**Actual:**
The comments provided by people above the "Meet Intellisense" are getting truncated at 400% zoom.

**Expected Result:**
All the content should be clearly visible at 400% zoom.

**User Impact:**
Low vision users who zoom the page for better visibility will find it difficult to view the content if text is truncated.

**Attachment:**
![A11y_Visual Studio Code Services_Home_Reflow](https://user-images.githubusercontent.com/97668775/149918321-fe0cafbe-965c-4d4a-83c4-c5487d3576a7.gif)

**Comment by gregvanl** [comment]
The text being truncated is Tweet verbatim used for promotion and not critical for user understanding or download workflow.

**Comment by poojapatil36** [comment]
@gregvanl it is bad experience and there is lot of white space where the text can be accommodated. Please don't close this bug.

**Comment by gregvanl** [comment]
Another consideration is that the website landing and docs pages are currently undergoing a redesign and will most likely remove this carousel of verbatim. We don't want to invest in fixing this issue right now.

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
Please find below attachment for reference.
Github tag: #Closed;
![Presentation3](https://user-images.githubusercontent.com/97668775/218081820-4cc8864b-bafc-492c-bb14-edf8deda7ad0.gif)
